sfs/mono/source.py

```diff
@@ -79,6 +79,54 @@ def point_velocity(omega, x0, n0, grid, c=None):
     return util.XyzComponents([v * o / r for o in offset])
 
 
+def point_dipol(omega, x0, n0, grid, c=None):
+    """Point source with dipole characteristics.
+
+    Parameters
+    ----------
+    omega : float
+        Frequency of source.
+    x0 : (3,) array_like
+        Position of source.
+    n0 : (3,) array_like
+        Normal vector (direction) of dipole.
+    grid : triple of numpy.ndarray
```

Comment:
I meant you should change `numpy.ndarray` to `array_like`!
I was only talking about the line before my comment (line 93).

You added `tripple` (which should be `triple` BTW) at many places where it doesn't belong!